app/site/_layouts/repo-report.liquid

```diff
@@ -155,6 +170,39 @@ layout: base
           </div>
           <p>{{ project.pull_requests_count }}</p>
         </div>
+        <div class="stat-container" id="project-age">
+          <div class="project-age-heading">
+            <h4>
+              <span>
+                <svg
+                  class="usa-icon"
+                  aria-labelledby="project-age"
+                  role="img"
+                >
+                  {% lucide "calendar" %}
+                </svg>
+              </span>
+              Age Of Project
+            </h4>
+          </div>
+          <p onclick="toggleDateAge(this)" style="cursor: pointer;">
```

Comment:
Ideally I would like it to be a bit clearer to the user that you can click the date section to get more information. The section should be underlined or something to make it seem like a hyperlink or button.